app/routes.js

```diff
@@ -13,27 +13,21 @@ module.exports = function(app) {
 	res.send(words[index]);
     });
 
-    app.get('/api/sessions', function(req, res) {
+    app.get('/api/sessions', function (req, res, next) {
```

Comment:
`next` can be called to pass control on to either the next "middleware" function or, if called with an argument, to the error handler.